Dockerfile.ci

```diff
@@ -1334,7 +1334,7 @@ ARG CASS_DRIVER_NO_CYTHON="1"
 # Build cassandra driver on multiple CPUs
 ARG CASS_DRIVER_BUILD_CONCURRENCY="8"
 
-ARG AIRFLOW_VERSION="2.5.0.dev0"
+ARG AIRFLOW_VERSION="2.7.0.dev0"
```

Comment:
Small thing I found - not that it matters, but worth updating.

Reply:
If the value doesn’t matter, maybe the default should be removed entirely instead.

Reply:
Yep. Good idea actually.

Reply:
I looked closer and it turned out it did not matter so much that after the version as been moved from setup.py, we stopped retrieving it correctly and the arg was always overwritten with `--build-arg AIRFLOW_VERSION=` - i.e. empty value. In case of CI image it is really only used for labeling and metadata (so no big deal) but yeah. I removed it from the image and rely on the one derived from `__init__.py` now + I added protection in case this will break in the future.